interview-questions.md

```diff
@@ -10,23 +10,23 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Your answer: A parameter goes next to a variable function and a argument goes with an if statement.
 
-  Researched answer: Parameters are properties of a function. Arguments are properties of a particular call to a function.
+  Researched answer: Parameters are properties of a function. Arguments are properties of a particular call to a function.You would use parameters as placeholders for an arugument in a function.A argument has to be a datatype recoginized by javascript and that data type is passed inside the parenthesis of a function invocation. The parameter and argument are different because the argument is using the parameter () to create an argument in console log. 
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer: Key values amd arrays. Both are optional.
+  Your answer: The built in methods are key values and arrays. Both are optional.
 
-  Researched answer:.map() have three built-in parameters - the value, index, and array. The index and array are both optional.
+  Researched answer: .map() is a higherorder function that has three built predefined parameters which are the value, index, and array. The index and array are both optional. The value is always required.
```

Comment:
When you have questions that can be answered quickly, fill the space with how you used the topic.